crates/foundry/evm/core/src/opts.rs

```diff
@@ -113,8 +124,9 @@ impl EvmOpts {
     }
 
     /// Returns the `revm::Env` configured with only local settings
-    pub fn local_evm_env(&self) -> revm::primitives::Env {
-        let mut cfg = CfgEnv::default();
+    pub fn local_evm_env(&self) -> EvmEnv<BlockT, TxT, HardforkT> {
+        // Non-exhaustive
```

Comment:
What does `non-exhaustive` refer to in this context?

Reply:
Improved comment here: https://github.com/NomicFoundation/edr/pull/901/commits/e4571e01e3f597bf60cda508725f73a18b93f107#diff-9a72ca000b61463d04f31261f2c2a116ca31dac021896b9f5149b184a21cda56R74